### PR TITLE
Adds initial firebase setup. Firebase is loading on main page display…

### PR DESCRIPTION
**Pull request by awllms**
…ing data from database. Remove vite eslint as it was causing errors. We should put it back on once the database is fully flesh out and we can resolve the errors

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,22 +1,65 @@
...
 import Home from './pages/Home';
+import db from '../src/utils/firebase';
 
 function App() {
   const [count, setCount] = useState(0);
+  const [entries, setEntries] = useState([]);
+  const [terms, setTerms] = useState({});
+  const [users, setUsers] = useState([]);
 
+  useEffect(() => {
```
Comment (sonyakim-dev) [inline src/App.jsx:13]: I heard it's better to use useQuery for fetching data than useEffect

Comment (awllms) [reply src/App.jsx:13]: I can use it but I'd like to get the prototype working with the live data first.

Comment (awllms) [reply src/App.jsx:13]: or that could be a good issue for someone once we finish.

File: src/App.jsx [hunk 2]
```diff
@@ -1,22 +1,65 @@
-import { useState } from 'react';
+import { collection, query, where, getDocs, terminate } from 'firebase/firestore';
+import { useState, useEffect } from 'react';
 import './App.css';
 import Home from './pages/Home';
+import db from '../src/utils/firebase';
```
Comment (sonyakim-dev) [inline src/App.jsx:5]: if you do `import {db} from './utils/firebase` then you don't need to do `db.db` on the below codes.

Comment (sonyakim-dev) [reply src/App.jsx:5]: oh, `import {db} from './utils/firebase` creates import error. changing `export default { db, auth };` to `export { db, auth };` in firebase.js will fix the import issue.

File: src/App.jsx (hunk 1 above)
Comment (sonyakim-dev) [reply src/App.jsx:13]: ok. when the page is rerendered, i can see multiple YRL. we can fix this on another pr

File: src/App.jsx (hunk 2 above)
Comment (awllms) [reply src/App.jsx:5]: That leads to a complaint from eslint:

No default export found in imported module

Comment (sonyakim-dev) [reply src/App.jsx:5]: yeah i got the issue and left the comment for fixing it! 😁

File: src/App.jsx (hunk 1 above)
Comment (awllms) [reply src/App.jsx:13]: I will fix the re-rendering. I'm opening a new pull tonight. I want to fix and add a few things.

File: src/App.jsx (hunk 2 above)
Comment (awllms) [reply src/App.jsx:5]: I fixed it! I will open a new pull request tonight.

**Pull request closed without merging**